source/mapping-rules/mapping-rule-options/mapping-rule-options.txt

```diff
@@ -1,8 +1,81 @@
+.. _rm-mapping-rule-options:
+
 ====================
 Mapping Rule Options
 ====================
 
+Mapping rule options are important options that help define the state
+of your MongoDB collection after a Migration.
+
+These options are defined when creating or editing a 
+:ref:`mapping rule <rm-mapping-rules>`.
+
+Mapping Rule Concepts
+---------------------
+
+Every mapping rule joins a source table to a destination collection. 
+You can create mapping rules from either a table or an entity.
+When creating a mapping rule you must select the other entity. For example:
+
+- If you select a table and choose to create a mapping rule you must choose the destination collection. 
+- If you select a collection and choose to create a mapping rule, you must select the source table.
+
+For :guilabel:`Embedded array` and :guilabel:`Embedded document` mapping rules, 
+you must also configure where you want the embedded data inserted into your document. 
+
+The table below provides a summary of the concepts associated with 
+mapping rule options and the mapping rules they apply to.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 50 25
+
+   * - Concept
+     - Description
+     - Applies To
+   * - Collection Name
+     - The destination collection where the relational rows of data are migrated.
+     - :guilabel:`New documents` 
+   * - Parent collection
+     - The destination collection where the relational rows of data are migrated.
+     - :guilabel:`Embedded documents` :guilabel:`Embedded array` 
+   * - Prefix
+     - A drop-down where you can choose where in the hierarchy the rule goes.
+     - :guilabel:`Embedded documents` :guilabel:`Embedded array` 
+   * - Field name
+     - A text field which indicates the field that contains the array or document.
```

Comment:
(wording)
Consider something like: Indicates the field containing the array or document.

Reply:
Updated.